src/components/table/helpers/mixin-tbody-row.js

```diff
@@ -90,7 +90,12 @@ export default {
     renderTbodyRowCell(field, colIndex, item, rowIndex) {
       // Renders a TD or TH for a row's field
       const h = this.$createElement
-      const hasDetailsSlot = this.hasNormalizedSlot(detailsSlotName)
+      const hasDetailsSlot =
+        this.hasNormalizedSlot(detailsSlotName) ||
+        this.computedFields.some(e => e.fieldDetailsSlotName)
```

Comment:
The cell based details should use a standard name, based on the field, rather than allowing aribtrary slot names (which have the potential to conflict with other slots in the table).

Please use a `boolean` for this property, and use the name `row-details(${field.key})`

Reply:
Otherwise, if we introduce a new slot name in the future for table, it could potentially conflict with users arbitrary slot names.

Reply:
Which is part of the reason we switched to the new naming convention for cell, header, and footer slots